kong/clustering/init.lua

```diff
@@ -205,15 +206,57 @@ local function fill_empty_hashes(hashes)
   end
 end
 
+
+--- detect '/v1/wrpc' endpoint
+--- if there is no '/v1/wrpc', fallback to websocket + json
```

Comment:
This documentation is not 100% accurate, it describes the goal, but not what this function does.
```suggestion
--- Return the highest supported Hybrid mode protocol version.
```

Also, does this needs to be inside `init.lua`? The CP shouldn't need this functionality.

Reply:
I consider moving it to `utils.lua`, but it may cause conflict with PR #8861.
So I want to do it in #8861, Do you think it is OK?

Reply:
Could it be moved to `wrpc_data_plane.lua`?

Reply:
I think it is not suitable for `wrpc_data_plane.lua`, because at this point we have not decided which module will be required.